appcenter-data/package.json

```diff
@@ -31,14 +31,14 @@
     }
   },
   "devDependencies": {
-    "babel-eslint": "^7.2.3",
-    "eslint": "^4.5.0",
-    "eslint-config-airbnb": "^15.1.0",
-    "eslint-plugin-import": "^2.7.0",
-    "eslint-plugin-jsx-a11y": "^6.0.2",
-    "eslint-plugin-react": "^7.2.1",
-    "jest": "^24.8.0",
-    "react": "^16.6.3",
+    "babel-eslint": "7.2.3",
```

Comment:
I am curious why its only Data, are other packages already pinned?

Reply:
It seemed to be the scope of the card, I can definitely add pinning for other packages as well. Would that go in this PR merging into mbaas-data or should it be a separate PR into develop?

Reply:
Can have 2 PRs, let's keep this one as is and make another for other modules in develop.